src/store/checkout-store.js

```diff
@@ -5,12 +5,20 @@ import StudentStore from './student-store';
 
 let checkouts = Object.create(null);
 let checkoutErrors = Object.create(null);
+let longTermCheckouts = Object.create(null);
 
 class CheckoutStore extends Store {
 
     getCheckouts() {
         return Object.keys(checkouts).map(key => checkouts[key]);
     }
+    getLongTermCheckouts() {
```

Comment:
https://codecov.io/gh/TheFourFifths/consus/src/61a2d98f7e04b11ee43c32e5254befcb3d39b35a/src/store/checkout-store.js

There should be tests for these methods.